testnet/stacks-node/src/config.rs

```diff
@@ -2358,6 +2359,9 @@ pub struct MinerConfig {
     pub wait_on_signers: Duration,
     /// Whether to mock sign in Epoch 2.5 through the .miners and .signers contracts. This is used for testing purposes in Epoch 2.5 only.
     pub pre_nakamoto_mock_signing: bool,
+    /// The minimum gap to wait between blocks in seconds. The value must be greater than or equal to 1 second because if a block is mined
+    /// within the same second as its parent, it will be rejected by the signers.
+    pub min_block_time_gap_secs: u64,
```

Comment:
Let's use ms here, since all the other times are in ms.

Reply:
I had tihs orignally but it seems weird to store ms when it compares against seconds in the timestamp. I can do this, but I would expect the timestamp to store ms at this point...I can still do if you tyihnk so, but will require an extra conversion step wen doing the comparison.

Reply:
Seconds was what I originally wanted to use, but I was convinced it should be milliseconds in the sync the other day. I don't really have a strong opinion on it. There are good arguments for both.

Reply:
HMMM....I guess from a user experience perspective AND for future proofing I will change to milliseconds...It will be a super small change if we want to modify the timestamp to ms precision down the line which arguably we should do anyway.